app/views/utilities/new.html.erb

```diff
@@ -0,0 +1,19 @@
+  <%= render "header", name: "NEW CATEGORY" %>
+<%= form_with model: @utility, url: user_utilities_path, class: "form" do |f| %>
+  <div class="field form__group first file_upload">
+    <div class="row">
+      <%= image_tag utility_icon(@utility, 100), class: "icon" %>
+      <div class="btn btn-image">
+        <i class="fa-solid fa-upload"></i>  Upload
```

Comment:
- [ ] Let's work on the upload feature in this form because it doesn't accept uploaded media on the live demo link 🙏